src/main/java/com/google/collegeplanner/data/Schedule.java

```diff
@@ -24,6 +24,13 @@ public Schedule() {
     sections = new ArrayList<Section>();
   }
 
+  public Schedule(Schedule copy) {
+    sections = new ArrayList<Section>();
```

Comment:
I think either `new ArrayList<>(copy.getSections());` or `Collections.copy(copy.getSections(), sections)` is preferred over manually iterating